dts/arm/st/l4/stm32l4.dtsi

```diff
@@ -300,6 +300,30 @@
 			label = "ADC_1";
 			#io-channel-cells = <1>;
 		};
+
+		lptim1: timers@40007c00 {
+			compatible = "st,stm32-timers";
+			clocks = <&rcc STM32_CLOCK_BUS_APB1 0x80000000>;
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40007c00 0x400>;
+			interrupts = <65 0>;
+			interrupt-names = "wakeup";
+			status = "disabled";
+			label = "LPTIM_1";
+		};
+
+		lptim2: timers@40009400 {
```

Comment:
Remove lptim2 if we don't have code to use it.

Reply:
ok